### ISSUE TITLE
Silently fails and does not generate docs

### ISSUE THREAD

**Issue by RhysC** [picklesdoc/pickles#202]
```
.\src\packages\Pickles.CommandLine.1.1.0\tools\pickles.exe `
	--feature-directory=.\src\My.Tests\ `
	--output-directory=.\documentation\ `
	--link-results-file=My.Tests.dll.output.xml `
	--test-results-format=xunit 
```

create a documentation dir and nothing else, no failure logs, nothing. ```$lastexitcode```is 1

**Comment by dirkrombauts** [comment]
Hi,

A couple of questions:

1. Does the problem always occur in your environment, or only with specific feature files?
2. Could you try to run pickles without including test results?
3. It would be most helpful if you could send me your feature files and test results file, but I appreciate that might be difficult because of IP reasons.

Dirk

**Comment by RhysC** [comment]
It was when test results were added, else docs were generated

**Comment by dirkrombauts** [comment]
OK, that's a clear indicator where to look. I'll try to look into it during the upcoming week, I have some "pickles time" set aside anyway because of the upcoming release of Visual Studio 2015.

**Comment by dirkrombauts** [comment]
Which version of xunit did you use to produce the test results? Version 2 perhaps?

**Comment by dirkrombauts** [comment]
I added a statement that should the command line version of Pickles fail, it will log the exception. I will make a proper 1.2 release later this week or early next week.

However, this is not going to do you much good in the default configuration, because it will only tell you that something went wrong. In order to tease a bit more information out of Pickles, you will need to adapt the NLog.config file that lives in the same directory with Pickles.exe. An example of an adapted config file is here:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <!--
  See http://nlog-project.org/wiki/Configuration_file
  for information on customizing logging rules and outputs.
   -->
    <targets>
        <target name="console" xsi:type="Console" />
        <target name="file" xsi:type="File" fileName="${basedir}/file.txt"
          layout="${longdate}|${level:uppercase=true}|${logger}|${message}|${exception}|${stacktrace}" />
    </targets>
    <rules>
        <logger name="*" minlevel="Info" writeTo="console" />
        <logger name="*" minlevel="Trace" writeTo="file" />
    </rules>
</nlog>
```

For more information on the variables allowed in the layout attribute, see https://github.com/nlog/NLog/wiki/Layout-renderers

**Comment by dirkrombauts** [comment]
The silent failure is fixed in #205

**Comment by dirkrombauts** [comment]
I know that the root problem (the failure itself) isn't solved, but for that I would need files to reproduce the error. Could you provide those please?
